skiko/src/awtMain/kotlin/org/jetbrains/skiko/SkiaLayer.awt.kt

```diff
@@ -20,7 +20,8 @@ actual open class SkiaLayer internal constructor(
     externalAccessibleFactory: ((Component) -> Accessible)? = null,
     private val properties: SkiaLayerProperties,
     private val renderFactory: RenderFactory = RenderFactory.Default,
-    private val analytics: SkiaLayerAnalytics = SkiaLayerAnalytics.Empty,
+    actual val pixelGeometry: PixelGeometry = PixelGeometry.UNKNOWN,
```

Comment:
As far as I understand, this can be changed in real time, and in the future we need to somehow react on these changes.

For example, when we use a tablet, that can happen often.

Reply:
Yep, I think that easiest way is to re-initialize a layer with different settings